spec/webmock/api_get_repo_pages.json

```diff
@@ -1 +1 @@
-{"message":"Not Found","documentation_url":"https://developer.github.com/v3"}
\ No newline at end of file
+{"message":"Not Found","documentation_url":"https://developer.github.com/v3"}
```

Comment:
Ah! Ok cool, looking through the history, it looks like it's trying to fetch data from this repo, which doesn't have Pages enabled so that'd explain it. Womp. Ideally we'd exercise both repo compat and the repo API stuff.